api/go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/ceph/ceph-csi/api
 
-go 1.22.2
+go 1.22.0
```

Comment:
This change should go to other commit where it was added not to this commit :)

Reply:
oh yea, I missed that.
it should have gone to `rebase: update kubernetes and go version` commit.